Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -341,6 +348,29 @@ def audit_formula_name
       openssl@1.1
     ].freeze
 
+    def audit_license
+      if formula.license.present?
+        if @spdx_data["licenses"].any? { |lic| lic["licenseId"] == formula.license }
+          return unless @online
+
+          user, repo = get_repo_data(%r{https?://github\.com/([^/]+)/([^/]+)/?.*}) if @new_formula
+          user ||= nil
+          repo ||= nil
+          return if user.nil?
```

Comment:
```suggestion
          return if user.blank?
```

(and same below)

Reply:
Hi @MikeMcQuaid, not sure if this works. It seems that `user` can default to `NilClass` as you mentioned but `NilClass` does not have a method `.blank?`

Reply:
Also, not sure why we use `.blank?` instead of `.nil?`. I understand that `.blank?` covers more cases such as an empty `Array` or empty `String` but in this case there is a specific case we are checking for and from my POV, it seems better to keep to a more specific scope when checking.

Reply:
@lionellloh `blank?` is defined on `nil` (it always returns `true`) through our inclusion of ActiveSupport. Generally it's nicer to use as `.nil?` because it's no slower and handles empty strings or arrays which we'd want to exclude.

Reply:
I see! I suspected it had something to do with ActiveSupport should have tried instead of just relying on the `irb`. Thanks, I will change the rest.

Reply:
Pro tip: use `brew irb` for Homebrew stuff!